features/upgrade/sdn/policy-upgrade.feature

```diff
@@ -78,3 +78,116 @@ Feature: SDN compoment upgrade testing
       | curl | -s | --connect-timeout | 5 | <%= cb.pod2ip %>:8080 |
     Then the step should fail
     And the output should not contain "Hello"
+
+
+  # @author asood@redhat.com
+  @admin
+  @upgrade-prepare
+  Scenario: Check the namespace networkpolicy for an application works well after upgrade - prepare
+    Given I switch to cluster admin pseudo user
+    When I run the :new_project client command with:
+      | project_name | test1 |
+    Then the step should succeed
+    When I run the :new_project client command with:
+      | project_name | test2 |
+    Then the step should succeed
+
+    When I use the "test1" project
+    Given I obtain test data file "networking/list_for_pods.json"
+    And I run the :create client command with:
+      | f | list_for_pods.json |
+    Then the step should succeed
+    Given 2 pods become ready with labels:
+      | name=test-pods |
+    And evaluation of `pod(1).ip` is stored in the :pod2ip clipboard
+    And evaluation of `pod(0).name` is stored in the :pod1 clipboard
+
+    Given I obtain test data file "rc/idle-rc-1.yaml"
+    And I run the :create client command with:
+      | f | idle-rc-1.yaml |
+    Then the step should succeed
+    Given 2 pods become ready with labels:
+      | name=hello-idle |
+    And evaluation of `pod(2).name` is stored in the :pod3 clipboard
+
+    When I use the "test2" project
+    Given I obtain test data file "networking/list_for_pods.json"
+    And I run the :create client command with:
+      | f | list_for_pods.json |
+    Then the step should succeed
+    Given 2 pods become ready with labels:
+      | name=test-pods |
+    And evaluation of `pod(4).name` is stored in the :pod5 clipboard
+
+    Given I obtain test data file "rc/idle-rc-1.yaml"
+    And I run the :create client command with:
+      | f | idle-rc-1.yaml |
+    Then the step should succeed
+    Given 2 pods become ready with labels:
+      | name=hello-idle |
+    And evaluation of `pod(6).name` is stored in the :pod7 clipboard
+
+    Given I have a project
```

Comment:
`Given I have a project`  should not be used.

Reply:
@zhaozhanqi I was still label the project without using 'Given I have a project' step so I can remove this step but what are the concerns using this step.

Reply:
this step will create new project but not use it in the following,  means it will create one non-used project.

Reply:
I do not mind removing this step as reduces number of steps in scenario. I do not think it tries to create a new project. If had attempted to create a new project the script would give an error - project already exits.